platform/android/export/export.cpp

```diff
@@ -778,6 +778,87 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	void _write_tmp_manifest(const Ref<EditorExportPreset> &p_preset, bool p_give_internet, bool p_debug) {
+		String manifest_header =
+				"<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
+				"<manifest xmlns:android=\"http://schemas.android.com/apk/res/android\"\n"
+				"    xmlns:tools=\"http://schemas.android.com/tools\">\n";
+
+		String manifest_screen_sizes = _get_screen_sizes_tag(p_preset);
+		String manifest_gles = _get_gles_tag();
+
+		Vector<String> perms;
+		_get_permissions(p_preset, p_give_internet, perms);
+		String manifest_permission;
+		for (int i = 0; i < perms.size(); i++) {
+			manifest_permission = manifest_permission.insert(0, "\t<uses-permission android:name=\"" + perms.get(i) + "\"/>\n");
+		}
+
+		String xr_metadata_text = "";
+		String fa_metadata_text = "";
+		String head_tracking = "";
+		String hand_tracking = "";
+		int xr_mode_index = p_preset->get("xr_features/xr_mode");
+		if (xr_mode_index == 1 /* XRMode.OVR */) {
+			String focus_awareness = bool_to_string(p_preset->get("xr_features/focus_awareness"));
+			xr_metadata_text = "<meta-data android:name=\"com.samsung.android.vr.application.mode\" android:value=\"vr_only\" />\n";
+			fa_metadata_text = "<meta-data android:name=\"com.oculus.vr.focusaware\" android:value=\"" + focus_awareness + "\"/>\n";
```

Comment:
Nit: on the previous line you used " />" but here use "/>". Please be consistent. (My personal preference is for the former.)